pkg/syncer/cnsoperator/util/util.go

```diff
@@ -188,3 +191,52 @@ func GetNetworkProvider(ctx context.Context) (string, error) {
 	return "", fmt.Errorf("could not find network provider field in configmap %q in namespace %q",
 		wcpNetworkConfigMap, kubeSystemNamespace)
 }
+
+// GetConfig is a wrapper function in syncer container to get the
+// config from vSphere Config Secret. If cluster ID is not provided
+// in the secret, then we read it from the immutable ConfigMap
+// which was created during csi controller initialization.
+func GetConfig(ctx context.Context) (*cnsconfig.Config, error) {
```

Comment:
`pkg/syncer/cnsoperator/util/` contains util for CNS Operator.
Can you move this util function at `pkg/syncer/util.go`